backend/tracim_backend/views/core_api/workspace_controller.py

```diff
@@ -242,10 +244,13 @@ def workspaces_members(
         """
         app_config = request.registry.settings["CFG"]  # type: CFG
         rapi = RoleApi(
-            current_user=request.current_user, session=request.dbsession, config=app_config
+            current_user=request.current_user,
+            session=request.dbsession,
+            config=app_config,
+            show_disabled_user=hapic_data.query.show_disabled_user == 1,
```

Comment:
hapic_data.query.show_disabled_user is already a boolean, no need conversion (already done  by WorkspaceMemberFilterQuery)